client/state/data-layer/wpcom/sites/gutenberg/index.js

```diff
@@ -31,7 +31,12 @@ const fetchGutenbergOptInData = ( action ) =>
 
 const setGutenbergOptInData = (
 	{ siteId },
-	{ editor_web: editor, opt_in: optIn, opt_out: optOut }
+	{
+		editor_web: editor,
+		opt_in: optIn,
+		opt_out: optOut,
+		in_editor_deprecation_group: inEditorDeprecationGroup,
```

Comment:
This needs plumbing through to reducer and a selector adding in order to retrieve it